src/olympia/stats/tests/test_commands.py

```diff
@@ -26,23 +26,29 @@
 
 
 class FixturesFolderMixin(object):
-    # You have to define these two values in your subclasses.
+    # You have to define these values in your subclasses.
     date = 'YYYY-MM-DD'
     source_folder = 'dummy'
     stats_source = 'dummy'
 
+    def get_tmp_hive_folder(self):
+        return os.path.join(hive_folder, self.id())
+
     def clean_up_files(self):
-        dirpath = os.path.join(hive_folder, self.date)
+        tmp_hive_folder = self.get_tmp_hive_folder()
+        dirpath = os.path.join(tmp_hive_folder, self.date)
         if os.path.isdir(dirpath):
             for name in os.listdir(dirpath):
                 os.unlink(os.path.join(dirpath, name))
             os.rmdir(dirpath)
+        if os.path.isdir(tmp_hive_folder):
+            os.rmdir(tmp_hive_folder)
```

Comment:
you can use `rm_stored_dir` from `storage_utils.py` to clear up multi-level folders/files.